esp/esp/django_settings.py

```diff
@@ -51,6 +51,7 @@
 # Must be unique for every site hosted
 CACHE_PREFIX="ESP"
 
+ALLOWED_HOSTS = ['*']
```

Comment:
Django seems to [think](https://docs.djangoproject.com/en/1.8/ref/settings/) this is a bad idea for security reasons, so we may want to have this set in `local_settings` per-set (and possibly have it default to *.learningu.org).  I _think_ our current apache and cache configurations on both *.learningu.org and esp.mit.edu protect us from malicious host-setting, but I wouldn't want to bet on it, and those configs change and are a bit more fragile than this setting.